kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/common/create-domain-job.sh

```diff
@@ -3,6 +3,13 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 #
 
+function exitIfError {
+  if [ "$1" != "0" ]; then
+    echo "$2"
+    exit $1
```

Comment:
Should echo/exit be replaced with "fail"?
Should this method be put in 'utilities'?

Reply:
This script is running in the job pod, and it does not use anything in utilities.